shell/platform/darwin/ios/ios_gl_context.h

```diff
@@ -13,28 +13,25 @@
 #include "flutter/fml/macros.h"
 #include "flutter/fml/platform/darwin/scoped_nsobject.h"
 #include "flutter/shell/common/platform_view.h"
-#include "ios_gl_render_target.h"
 
 namespace flutter {
 
 class IOSGLContext {
  public:
-  IOSGLContext();
+  IOSGLContext(fml::scoped_nsobject<EAGLSharegroup> sharegroup);
```

Comment:
nit: might be nice to have a default for this so that callers don't have to explicitly pass in `nullptr` when they don't want a sharegroup.

Reply:
I'm doing a small change to switch to xster's suggestion of having a factory method instead. Thinking of having a `MakeSharedContext()` factory, which I've decided I prefer because it stops the caller from needing to know about sharedgroups and can treat the IOSGLContext a bit more opaquely.